### PR TITLE
[WIP] Dummy provisioner for testing purpose

### PR DESCRIPTION
**Pull request by ahsio**
Once done, this fix should address https://github.com/lafourchette/prototype/issues/5
- [x] Add a simple factory to build provisioners
- [x] Add a dummy provisioner that stores data in a file on the target installation directory 
- [ ] This new dummy provisioner should be used by default <b>[In progress]</b>
- [ ] Write unit tests <b>[In progress]</b>
- [ ] [Extra] Code refactoring <b>[In progress]</b>

### CODE REVIEW

**Comment by dav-m85** [comment]
Looks good so far, keep it up :)

**Comment by ahsio** [comment]
@dav-m85 I think you should consider using this git pre-commit hook https://gist.github.com/ahsio/7768689.
Which will prevent you from pushing PSR-2 non-compliant code to your remote repository.
And feel free to contribute to the script!

**Comment by ahsio** [comment]
@dav-m85 I think you should consider using this git pre-commit hook https://gist.github.com/ahsio/7768689. Which will prevent you from pushing non PSR-2 compliant code to your remote repository. And feel free to contribute to the script!

File: src/LaFourchette/Provisioner/Dummy.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        "identifier_1" : "value_1",
+        "identifier_2" : "value_2",
+        "identifier_x" : "value_x",
+    }
+}
+EOS;
+
+        $this->sendfile($vm, 'dataFile', $dataFile);
+    }
+}
```
Comment (ahsio) [inline src/LaFourchette/Provisioner/Dummy.php]: @dav-m85 I used @todo tags to ask questions about the way this dummy provisioner should work! Can you please address them :)

**Pull request closed without merging**